### PR TITLE
Add unittest for net6 bug

### PR DESCRIPTION
**Pull request by synercoder**
### Prerequisites

- [X] I have written a descriptive pull-request title
- [X] I have verified that there are no overlapping [pull-requests](https://github.com/SixLabors/ImageSharp/pulls) open
- [X] I have verified that I am following the existing coding patterns and practice as demonstrated in the repository. These follow strict Stylecop rules :cop:.
- [X] I have provided test coverage for my change (where applicable)

### Description
A regression test for #1765, #1704 & #1707 as proposed to @antonfirsov in #1765.

### CODE REVIEW

**Comment by synercoder** [comment]
@antonfirsov could you provide me with a clue where I went wrong? I added the file, added the test. Test works on my end... But it seems that automated build can't find the file.

**Comment by synercoder** [comment]
Test running again... just thought that it might be a capital letter in path issue might be. Windows doesn't care... but Ubuntu might...

**Comment by synercoder** [comment]
> @antonfirsov could you provide me with a clue where I went wrong? I added the file, added the test. Test works on my end... But it seems that automated build can't find the file.

I know where I went wrong... it was indeed the uppercase/lowercase issue in the path.

File: tests/ImageSharp.Tests/TestImages.cs [hunk 1]
```diff
@@ -111,6 +111,9 @@ public static class Png
             // Issue 935: https://github.com/SixLabors/ImageSharp/issues/935
             public const string Issue935 = "Png/issues/Issue_935.png";
 
+            // Issue 1765: https://github.com/SixLabors/ImageSharp/issues/1765
+            public const string Issue1765 = "Png/issues/Issue_1765.png";
```
Comment (antonfirsov) [inline tests/ImageSharp.Tests/TestImages.cs:115]: Can we rename this to `Issue1765_Net6DeflateStreamRead` to have a reminder what is the issue about?

Comment (synercoder) [reply tests/ImageSharp.Tests/TestImages.cs:115]: I don't see why not ;) ill rename somewhere today

Comment (synercoder) [reply tests/ImageSharp.Tests/TestImages.cs]: Renamed the file itself & the constant

**Review by antonfirsov — APPROVED** [review]
Thank you!

**Pull request merged**